Lombiq.Tests.UI/Docs/Troubleshooting.md

```diff
@@ -20,6 +20,7 @@
     3. Run `dotnet dev-certs https --trust` and accept the security warning that pops up. (This is why we can't automate all of this.)
     4. Now `dotnet dev-certs https --check --verbose` should display "A valid certificate was found."
 - If you have retries configured for the tests and they've been running for a long time without any visible progress then most possibly all of them are failing. This can happen e.g. if the setup or initialization steps are failing. However, by default the `OrchardCoreUITestExecutorConfiguration.FastFailSetup` configuration will quickly fail all affected tests if a common setup operation is failing, without retrying it for all tests.
+- If tests fail at at the start with the exception screen saying ``InvalidOperationException: Unable to resolve service for type 'Microsoft.AspNetCore.Identity.ILookupNormalizer' while attempting to activate 'Microsoft.AspNetCore.Identity.RoleManager`1[OrchardCore.Security.IRole]'.``, add the _Lombiq.Tests.UI.Shortcuts_ dependency to your web project.
```

Comment:
Use a code block for the technical project name:
```suggestion
- If tests fail at the start with the exception screen saying ``InvalidOperationException: Unable to resolve service for type 'Microsoft.AspNetCore.Identity.ILookupNormalizer' while attempting to activate 'Microsoft.AspNetCore.Identity.RoleManager`1[OrchardCore.Security.IRole]'.``, add the `Lombiq.Tests.UI.Shortcuts` dependency to your web project.
```

Reply:
Why does referencing `Shortcuts` help?

Reply:
I don't know, @DemeSzabolcs do you know?

I merely relayed the instructions [in Teams](https://teams.microsoft.com/l/message/19:525a1a3656c845ae92c5e2c0d3617cc7@thread.skype/1648062552986?tenantId=959ede5a-38e9-44e6-adc8-85553beb8220&groupId=d6a5d775-6fb9-41dc-a7bb-481d4f884046&parentMessageId=1648062552986&teamName=Lombiq%20Developers&channelName=General&createdTime=1648062552986&allowXTenantAccess=false) so it's more discoverable. (Because my first instinct when something crashes is to google the error rather than to search Teams. This should get cached eventually so it will appear in the search next time we have this problem.)

Reply:
I don't know, but from my original Teams post it seems like that shortcuts were turned on. 
![image](https://github.com/Lombiq/UI-Testing-Toolbox/assets/80963259/09f91c1b-45ea-4b35-8abf-fbf1d0dabf86)

There were a lot of changes in UI testing toolbox since then, but I guess it's needed now because this became the default  (shortcuts turned on) or something like that.

Reply:
Shouldn't the shortcuts be a dependency then? It's quite confusing that you can reference `Lombiq.Tests.UI` by itself, and then some of your tests fail, because you **also** need `Lombiq.Tests.UI.Shortcuts`.

Reply:
It shouldn't be a required dependency, this is a bug for sure. I've added a separate issue issue to address it: OSOE-626.

Reply:
Here's the GH issue link: https://github.com/Lombiq/UI-Testing-Toolbox/issues/279.